mw-webapp/src/component/select/option/Option.tsx

```diff
@@ -0,0 +1,45 @@
+
+/**
+ * Option element for a select component
+ */
+export interface OptionType {
+  /**
+   * Option id. Should be unique
+   */
+  id: string;
+  /**
+   * Option value
+   */
+  value: string;
+  /**
+   * Option`s visible text
+   */
+  text: string;
+}
+/**
+* Option component
+*/
+interface OptionProps{
+  /**
+   * Unique value
+   */
+  key: string;
+  /**
+   * Option`s value
+   */
+  value: string;
+  /**
+   * Inner option text
+   */
+  text: string;
+}
+export const Option: React.FC<OptionProps> = (props: OptionProps) => {
```

Comment:
All public members (exported variables, functions, types, etc) should be documented. Check everywhere in PR

Reply:
got it